File: tournaments/waterTubes/waterTubes.cpp
int waterTubes(std::vector<int> water, std::vector<int> flowPerMinute) {
  int result = 0;

  for (int i = 0; i < water.size(); i++) {
    int minutes = water[i] / flowPerMinute[i];
    if (water[i] % flowPerMinute[i] != 0) {
      minutes++;
    }
    if (result < minutes) {
      result = minutes;
    }
  }
  return result;
}

File: tournaments/waterTubes/waterTubes.js
function waterTubes(water, flowPerMinute) {
    let result = 0;
    
    for (let i = 0; water.length > i; ++i) {
        let minutes = ~~(water[i] / flowPerMinute[i]);
        
        if (water[i] % flowPerMinute[i])
            ++minutes;
        if (result < minutes)
            result = minutes;
    }
    
    return result;
}
